run/pubsub/main.go

```diff
@@ -50,6 +50,7 @@ func main() {
 // PubSubMessage is the payload of a Pub/Sub event.
 // See the documentation for more details:
 // https://cloud.google.com/pubsub/docs/reference/rest/v1/PubsubMessage
+// byte slice unmarshalling handles base64 decoding
```

Comment:
Nit: capitalize/use a period for a sentence comment. go/go-style/decisions#comment-sentences